### ISSUE TITLE
部署qwen-vl的自定义模型时，哪怕选择了qwen-vl-chat，页面风格还是大语言模型的风格，而不是多模态有图片上传的

### ISSUE THREAD

**Issue by xs818818** [xorbitsai/inference#1032]
### To Reproduce
To help us to reproduce this bug, please provide information below:

1. Your Python version.
2. The version of xinference you use.
3. Versions of crucial packages.
4. Full stack of the error.
5. Minimized code to reproduce the error.

**Comment by aresnow1** [comment]
自定义还没支持选择 vl 的功能，近期会开发支持

**Comment by hainaweiben** [comment]
Actually, I tried downloading the qwen-vl-chat model, and upon the initial launch, the displayed interface was llama-2-7b-chat. However, when I reran xinference-local, the interface appeared normal.
And I noticed that when downloading the qwen-vl-chat model, the llama-2 model was also downloaded simultaneously.

The running model interface of the qwen-vl model is normal.
![image](https://github.com/xorbitsai/inference/assets/112739514/4b931b7e-e09f-4078-b32f-74292d964260)
